chia/wallet/wallet_node.py

```diff
@@ -364,6 +364,9 @@ async def complete_blocks(self, header_blocks: List[HeaderBlock], peer: WSChiaCo
         if self.wallet_state_manager is None:
             return None
         header_block_records: List[HeaderBlockRecord] = []
+        trusted = False
+        if self.server is not None and self.server.is_trusted_peer(peer, self.config["trusted_peers"]):
+            trusted = True
```

Comment:
those 3 lines are just
```python
trusted = self.server is not None \
    and self.server.is_trusted_peer(peer, self.config["trusted_peers"])
```
(In fact the `is not None` is also redundant if we#re assuming `self.server` to not switch type completely)